modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/marketplace-apps-categories/ddm-template.ftl

```diff
@@ -1,7 +1,114 @@
-<#if (AssetVocabulary_7784259.getData())??>
-	<div class="color-neutral-3 flex flex-wrap font-size-paragraph-small solution-category-container">
-		<div class="bg-neutral-8 border-radius-small mb-1 mr-1 px-1 solution-category">
-			${AssetVocabulary_7784259.getData()}
-		</div>
+<style>
+	.app-container {
+		font-size: MEDIUM;
```

Comment:
Could we specify the size? medium could be anything

```suggestion
		font-size: MEDIUM;
```

Reply:
I can remove it. It doesn't affect anything, I just got it from figma